apps/music/js/metadata/vorbis_picture.js

```diff
@@ -0,0 +1,132 @@
+/* global BlobView */
+/* exported VorbisPictureComment */
+'use strict';
+
+var VorbisPictureComment = (function() {
+  /**
+   * Read the picture frame from the block blob
+   */
+  function readPicFrame(view) {
+    // Source of the structure:
+    // https://xiph.org/flac/format.html#metadata_block_picture
+    // http://flac.sourceforge.net/format.html#metadata_block_picture
+    //
+    var kind = view.readUnsignedInt();
+    if (kind != 3) {
+      return null;
+    }
+    var len = view.readUnsignedInt();
+    var mimetype = view.readASCIIText(len);
+    if (mimetype == '-->') {
+      // XXX we don't support yet URL
+      console.error('URL for cover art are not supported');
```

Comment:
Nit: `s/are/is/`, and maybe add something about this being from a Vorbis Comment.